security/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+services:
+  mysql:
+    image: mysql:latest
+    container_name: db
+    environment:
+      MYSQL_ROOT_PASSWORD: 1234
+      MYSQL_DATABASE: security
+    networks:
+      - net
+    ports:
+      - "3306:3306"
+  java-app:
+    image: security
+    container_name: app
+    depends_on:
+      - mysql
+    networks:
+      - net
+    ports:
+      - "8080:8080"
```

Comment:
Services should have different ports from each other.
This port would clash with the chat service port as both are 8080